src/DataTypes/Serializations/SerializationFixedString.cpp

```diff
@@ -183,14 +185,17 @@ static inline bool tryRead(const SerializationFixedString & self, IColumn & colu
 }
 
 
-void SerializationFixedString::deserializeTextEscaped(IColumn & column, ReadBuffer & istr, const FormatSettings &) const
+void SerializationFixedString::deserializeTextEscaped(IColumn & column, ReadBuffer & istr, const FormatSettings & settings) const
```

Comment:
You should also check for all `deserializeTextEscaped` methods in all serializations, not only for String/FixedString, they all can use `readEscapedStringInto` or have own implementation relying only on  `\n` at the end

Reply:
does that include serializations like `serializationBool`? I see it calls `deserializeTextEscaped` but with it's own implementation, without any calls to `readEscapedStringInto`.

https://github.com/ClickHouse/ClickHouse/blob/5d6bc6f566f09d1cd2ecf414bbb74222ffffa6bd/src/DataTypes/Serializations/SerializationBool.cpp#L241-L247